Documentation/Makefile

```diff
@@ -148,6 +151,12 @@ live-preview: builder-image ## Build and serve the documentation locally.
 			$(DOCS_BUILDER_IMG) \
 		sphinx-autobuild --open-browser --host 0.0.0.0 --port $(DOCS_PORT) $(SPHINX_OPTS) --ignore *.swp -Q . _preview
 
+update-redirects: builder-image ## Build and serve the documentation locally.
+	@echo "$$(tput setaf 2)Writing redirects$$(tput sgr0)"
+	$(QUIET)$(DOCKER_CTR) \
+		$(DOCS_BUILDER_IMG) \
+		sphinx-build -M rediraffewritediff . _preview
```

Comment:
Could you please mention the redirects in https://docs.cilium.io/en/latest/contributing/docs/docsframework/#auto-generated-contents and/or https://docs.cilium.io/en/latest/contributing/docs/docsframework/#tweaks-and-tools? Not blocking for this PR, can be a follow-up.

Reply:
Do you think I should update the `check` target to run `update-redirects`? Technically this check already happens as part of the website build, so it's somewhat redundant, but It could be useful.

Reply:
Why not, but we should probably avoid running it twice in CI. Maybe gating it with `SKIP_BUILD`, since it's part of the build?